ArcGISRuntimeSDKQt_CppSamples/Scenes/FeatureLayerExtrusion/FeatureLayerExtrusion.cpp

```diff
@@ -79,32 +78,35 @@ void FeatureLayerExtrusion::componentComplete()
           this));
   scene->setBaseSurface(surface);
   scene->operationalLayers()->append(m_featureLayer);
-  m_sceneView->setArcGISScene(scene);
 
-  const Point m_lookAtPoint = Point(-10974490, 4814376, 0, SpatialReference::webMercator());
-  const double distance = 20000000;
-  OrbitLocationCameraController* m_orbitController = new OrbitLocationCameraController(m_lookAtPoint, distance, this);
+  // set initial viewpoint
+  const double distance = 12940924;
+  const Point lookAtPoint = Point(-99.659448, 20.513652, distance, SpatialReference::wgs84());
```

Comment:
Change to `const Point lookAtPoint(-99.659448, 20.513652, distance, SpatialReference::wgs84());`